src/libraries/System.Private.CoreLib/src/System/Environment.iOS.cs

```diff
@@ -14,6 +15,18 @@ public static partial class Environment
 #if !TARGET_MACCATALYST
         // iOS/tvOS aren't allowed to call libproc APIs so return 0 here, this also matches what we returned in earlier releases
         public static long WorkingSet => 0;
+
+        /// <summary>
+        /// Get the CPU usage, including the process time spent running the application code, the process time spent running the operating system code,
+        /// and the total time spent running both the application and operating system code.
+        /// </summary>
+        [SupportedOSPlatform("maccatalyst")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        public static ProcessCpuUsage CpuUsage
+        {
+            get { throw new PlatformNotSupportedException(); }
```

Comment:
Alternatively we could just return time values of 0 rather than throwing. I don't have a strong opinion either way.

Reply:
Returning 0s might also be more consistent with a case where GetCpuUtilization fails?

Reply:
Process throws but I am seeing Environment usually return default value. I'll change it as you suggested. I don't have any strong opinion either.

Reply:
Done!